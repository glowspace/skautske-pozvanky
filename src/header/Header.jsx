import React from 'react';
import styled from 'styled-components';
import { BigText } from '../styled/Styled';

const HeaderStyled = styled.div`
  display: flex;
  justify-content: space-between;
  margin-bottom: 2rem;
  padding-left: 0.3rem;
  box-shadow: rgba(0, 0, 0, 0.75) 0px 2px 5px 0px;
  font-family: skautbold, sans-serif;
  font-weight: normal;
  background-color: #a0067d;
  @media print {
    display: none;
  }
`;
const Header = () => {
  return (
    <HeaderStyled>
      <BigText color="white">Generátor pozvánek</BigText>
    </HeaderStyled>
  );
};

export default Header;
