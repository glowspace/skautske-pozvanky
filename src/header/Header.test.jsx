import React from 'react';
import Header from './Header';

test('header', () => {
  expect(<Header />).toMatchSnapshot();
});
