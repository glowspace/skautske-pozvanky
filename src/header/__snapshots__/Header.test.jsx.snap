// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header 1`] = `<Header />`;
